### ISSUE TITLE
MAUI App Scaling Issue on Windows

### ISSUE THREAD

**Issue by devcrux** [dotnet/maui#23065]
### Description

I am in the process of submitting a .net maui app to the Windows Store and it kept getting rejected for the reason below: 

---------------------------------------
Products must be fully accessible on devices using common resolution scaling values. 
The issue was observed on the following devices running OS build 22621.1105  
Observed On: Dell Inspiron 13-5379 
Tested On: Dell Inspiron 13-5379
Resolution &amp; Scaling Value: [1920x1080 resolution and 150%scaling percentage] 
---------------------------------------- 

So basically, the app works fine on 1920x1080 resolution at 100% scale but when you change the scale to 150% it seems that the controls scale out of proportion.

I checked the IWindows.cs in the Maui repo to see if I can restrict the display density but there is no such public method.

### Version with bug

8.0.3 GA

### Is this a regression from previous behavior?

Not sure, did not test other versions

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

**Comment by RoiChen001** [comment]
Can't repro this issue at Windows platform on the latest 17.11.0 Preview 2.0(8.0.60).
